shared/src/main/scala/com/raquo/domtypes/defs/tags/MathMlTagDefs.scala

```diff
@@ -0,0 +1,334 @@
+package com.raquo.domtypes.defs.attrs
+
+import com.raquo.domtypes.common.{MathMLTagType, TagDef}
+
+
+object MathMLTagDefs {
+
+    val defs: List[TagDef] = List(
```

Comment:
Please sort this list alphabetically by tag name (and same with the upcoming attr list – by attr name)